website/docs/r/is_bare_metal_server.markdown

```diff
@@ -97,12 +97,13 @@ Review the argument references that you can specify for your resource.
 - `name` - (Optional, String) The bare metal server name.
 
   -> **NOTE:**
-    a bare metal server can take up to 30 mins to clean up on delete, replacement/re-creation using the same name may return error
+    A bare metal server can take up to 30 mins to clean up on delete, replacement/re-creation using the same name may return error
 
-- `network_interfaces` - (Optional, List) The additional network interfaces to create for the bare metal server to this bare metal server. Use `ibm_is_bare_metal_server_network_interface` &  `ibm_is_bare_metal_server_network_interface_allow_float` resource for network interfaces.
+- `network_interfaces` - (Optional, List) The additional network interfaces to create for the bare metal server to this bare metal server. Use `ibm_is_bare_metal_server_network_interface` & `ibm_is_bare_metal_server_network_interface_allow_float` resource for network interfaces.
 
   ~> **NOTE:**
-    creating network interfaces both inline with `ibm_is_bare_metal_server` & as a separate `ibm_is_bare_metal_server_network_interface` resource, will show change alternatively on both resources, to avoid this use `ibm_is_bare_metal_server_network_interface` for creating network interfaces.
+    IBM Cloud terraform provider currently provides both a standalone `ibm_is_bare_metal_server_network_interface` resource and a `network_interfaces` defined in `ibm_is_bare_metal_server` resource to provision a network interface for bare metal. </br>
+    Creating network interfaces both inline with `ibm_is_bare_metal_server` & as a separate `ibm_is_bare_metal_server_network_interface` resource, will show change alternatively on both resources, to avoid this; Use `ibm_is_bare_metal_server_network_interface` as recommended resource for creating network interfaces.
```

Comment:
Sorry, I still don't know what "will show change alternatively on both resources" means.

Reply:
Kindly check the latest changes.

Reply:
The problem is: "will show change alternatively on both resources". I'm not sure what this means.